osi_trafficsign.proto

```diff
@@ -960,216 +960,2890 @@ message TrafficSign
                 // See e.g.:
                 //
                 TYPE_TRAFFIC_LIGHT_GREEN_ARROW = 92;
+
```

Comment:
I guess the inconsistency is "TYPE_DANGER_FALLING_ROCKS" , "TYPE_SNOW_OR_ICE", "TYPE_ROAD_WORK", "TYPE_WARNING_ROAD_NARROWING_LEFT " maybe it can just be altered in the 4.0.0